common/errors.js

```diff
@@ -146,29 +154,39 @@
              * @type {string}
              * @desc   Error message status; acts as Title text for error dialog
              */
-            this.status = messageMap.unauthorizedErrorCode;
+            this.status = null;
 
             /**
              * @type {string}
              * @desc   Error message
              */
-            this.message = errorMessages.unauthorizedAssetRetrieval;
+            this.message = null;
 
             /**
              * @type {string}
              * @desc Action message to display for click of the OK button
              */
-            this.errorData.clickActionMessage = messageMap.clickActionMessage.loginOrDismissDialog;
+            this.errorData.clickActionMessage = null;
 
             /**
              * @type {boolean}
              * @desc Set true to dismiss the error modal on clicking the OK button
              */
             this.clickOkToDismiss = true;
+
+            if (code == 401) {
```

Comment:
Isn't it better to create two different error types if these two cases have completely different status, message, and clickActionMessage?